viktor_dev_tools/cli.py

```diff
@@ -5,12 +5,14 @@
 """
 import subprocess
```

Comment:
please make sure to add any new functions to click, such that the users is prompted all options